docker/docker-compose.yml

```diff
@@ -27,14 +27,13 @@ services:
     deploy: *default-deploy
     volumes:
       - influxdb:/var/lib/influxdb/
-    healthcheck:
-      test: ["CMD-SHELL", "influx -execute 'quit'"]
-      interval: 5s
-      timeout: 5s
-      retries: 5
-      start_period: 10s
+      - "./influxdb/influxdb.conf:/etc/influxdb/influxdb.conf:ro"
+      - "./influxdb/setup-influxdb.sh:/docker-entrypoint-initdb.d/setup-influxdb.sh"
```

Comment:
Might be read-only too?
I am not sure, but maybe build a custom image?

Reply:
Initially I thought I would have to do a separate image but it actually worked out using the stock image. I would rather not have to add an additional image to maintain if we don't need to. If we find that we need to make a lot of custom changes in the future we could revisit.